lib/elixir/src/elixir_parser.yrl

```diff
@@ -726,9 +727,12 @@ to_block(Other) -> {'__block__', [], reverse(Other)}.
 
 %% Errors
 
+throw([Line, _, _], Error, Token) ->
+  throw({error, {Line, ?MODULE, [Error, Token]}});
 throw(Line, Error, Token) ->
```

Comment:
Same here, it feels like it would be a bug if we are still passing line as integer.